### PR TITLE
Replace Nokia firebase device with Samsung

### PR DESCRIPTION
**Pull request by kiryldz**
### Summary of changes

It seems that Nokia device is extremely flaky based on our instrument tests, let's try to use Samsung.

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `./gradlew apiDump` to update generated api files, if there's public API changes, otherwise the `verify-kotlin-binary-compatibility` CI will fail.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by Chaoba — APPROVED** [review]

**Review by ank27 — APPROVED** [review]

**Comment by ank27** [comment]
lets try this, our tests now runs for 2hrs, and then complain about flaky test 😞

**Comment by yunikkk** [comment]
> It seems that Nokia device is extremely flaky based on our instrument tests, let's try to use Samsung. 

would be interesting though what exactly is flacky, are we 100% sure it's some hidden issue with certain OS version? 
Would be nice to submit ticket to evaluate it back when we'll have tools to debug flacky tests (screenshots, logs, kaspresso). Mind adding one?

**Review by yunikkk — APPROVED** [review]

File: sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt [hunk 1]
```diff
@@ -961,7 +961,7 @@ class ViewAnnotationTest(
   }
 
   private companion object {
-    const val VIEW_PLACEMENT_DELAY_MS = 1000L
+    const val VIEW_PLACEMENT_DELAY_MS = 3_000L
```
Comment (yunikkk) [inline sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt:964]: Why?

Comment (kiryldz) [reply sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt:964]: perhaps 1 sec is not enough to load a style and position view annotations on CI

**Comment by kiryldz** [comment]
Closing in favor of https://github.com/mapbox/mapbox-maps-android/pull/1176

**Pull request closed without merging**